src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/OidcngEntityType.php

```diff
@@ -111,7 +111,6 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'entry_options' => [
                         'attr' => [
                             'data-parsley-redirecturis' => null,
-                            'data-parsley-uri' => null,
```

Comment:
Should this also be removed in the OidcEntity ?

Reply:
Yes, good point!